test/hotspot/jtreg/compiler/runtime/Test8168712.java

```diff
@@ -24,6 +24,7 @@
 /**
  * @test
  * @requires vm.debug
+ * @requires vm.hasDTrace
```

Comment:
DTrace is not critical to this test. IIUC the original fix and test had an issue when combined with DTrace hence the test enabled it to check it was fixed by a further patch. So we actually want to run this test regardless. In which case we need to add another test section for when `!vm.hasDTrace` that doesn't pass the DTrace flag.

Reply:
Right. Added, see new commit.